Composer/packages/client/src/pages/design/DesignPage.tsx

```diff
@@ -564,9 +572,15 @@ const DesignPage: React.FC<RouteComponentProps<{ dialogId: string; projectId: st
       regEx: '',
       triggerPhrases: '',
     };
-    if (dialogId) {
+    const dialog = dialogs.find((d) => d.id === dialogId);
+    if (dialogId && dialog) {
```

Comment:
nit: ```if(dialog)``` is sufficient

Reply:
need to check dialogId because of createTrigger(dialogId, formData): Argument of type 'string | undefined' is not assignable to parameter of type 'string'.